builtin/blame.c

```diff
@@ -319,18 +319,18 @@ static const char *format_time(timestamp_t time, const char *tz_str,
 	return time_buf.buf;
 }
 
-#define OUTPUT_ANNOTATE_COMPAT	001
-#define OUTPUT_LONG_OBJECT_NAME	002
-#define OUTPUT_RAW_TIMESTAMP	004
-#define OUTPUT_PORCELAIN	010
-#define OUTPUT_SHOW_NAME	020
-#define OUTPUT_SHOW_NUMBER	040
-#define OUTPUT_SHOW_SCORE	0100
-#define OUTPUT_NO_AUTHOR	0200
-#define OUTPUT_SHOW_EMAIL	0400
-#define OUTPUT_LINE_PORCELAIN	01000
-#define OUTPUT_COLOR_LINE	02000
-#define OUTPUT_SHOW_AGE_WITH_COLOR	04000
+#define	OUTPUT_ANNOTATE_COMPAT	(1<<0)
```

Comment:
Sorry, I should have been clearer: I meant to align the _values_ (i.e. `1<<0`, `1<<1`, etc), with spaces, and _not_ the keys. There should not be more than one single space between the `#define` and the `OUTPUT_ANNOTATE_COMPAT` and friends.